frontend/fingertips-frontend/components/organisms/LineChart/lineChartHelpers.ts

```diff
@@ -1,22 +1,43 @@
 import { HealthDataForArea } from '@/generated-sources/ft-api-client';
 import Highcharts, { SymbolKeyValue } from 'highcharts';
+import { ChartColours } from '@/lib/chartHelpers/colours';
 
 export function generateSeriesData(
   data: HealthDataForArea[],
   symbols: SymbolKeyValue[],
-  benchmarkData?: HealthDataForArea
-): Highcharts.SeriesLineOptions[] {
-  const seriesData = data.map<Highcharts.SeriesLineOptions>((item, index) => ({
-    type: 'line',
-    name: `${item.areaName}`,
-    data: item.healthData.map((point) => [point.year, point.value]),
-    marker: {
-      symbol: symbols[index % symbols.length],
-    },
-  }));
+  chartColours: ChartColours[],
+  benchmarkData?: HealthDataForArea,
+  showConfidenceIntervalsData?: boolean
+) {
+  const seriesData: Highcharts.SeriesOptionsType[] = data.flatMap(
+    (item, index) => {
+      const lineSeries: Highcharts.SeriesOptionsType = {
+        type: 'line',
+        name: `${item.areaName}`,
+        data: item.healthData.map((point) => [point.year, point.value]),
+        marker: {
+          symbol: symbols[index % symbols.length],
+        },
+        color: chartColours[index % chartColours.length],
+      };
+
+      const confidenceIntervalSeries: Highcharts.SeriesOptionsType = {
+        type: 'errorbar',
+        name: `${item.areaName}`,
+        data: item.healthData.map((point) => [
+          point.year,
+          point.lowerCi,
+          point.upperCi,
+        ]),
+        visible: showConfidenceIntervalsData,
```

Comment:
Can we do anything with the styling of the error bars? Compare the current implementation:
![Screenshot 2025-02-14 at 17 00 01](https://github.com/user-attachments/assets/dc2cb80b-d585-4610-955d-ac7069e56d88)

With the designs:
![Screenshot 2025-02-14 at 17 00 11](https://github.com/user-attachments/assets/b89d4975-642c-43e8-8d85-f3caab2fd4fe)

I think we need to try and make the error bars the correct shade of grey, make the horizontal lines narrower and possibly make all lines a bit thicker.

Reply:
Looks like there's a [`color` property](https://api.highcharts.com/highcharts/plotOptions.errorbar.color), a [`whiskerLength` property](https://api.highcharts.com/highcharts/plotOptions.errorbar.whiskerLength) and a [`lineWidth` property](https://api.highcharts.com/highcharts/plotOptions.errorbar.lineWidth) that should do the trick.

See https://www.highcharts.com/docs/chart-and-series-types/error-bar-series#styling-the-error-bars & https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/highcharts/plotoptions/error-bar-styling/.

Reply:
Something like this looks reasonable to me:
```suggestion
        type: 'errorbar',
        name: `${item.areaName}`,
        data: item.healthData.map((point) => [
          point.year,
          point.lowerCi,
          point.upperCi,
        ]),
        visible: showConfidenceIntervalsData,
        color: '#B1B4B6',
        whiskerLength: '20%',
        lineWidth: 2,
```

Reply:
Thanks